### PR TITLE
fix: add assets to releases

### PR DESCRIPTION
**Pull request by stebenz**
### Definition of Ready

- [x] I am happy with the code
- [ ] Short description of the feature/issue is added in the pr description
- [ ] PR is linked to the corresponding user story
- [ ] Acceptance criteria are met
- [ ] All open todos and follow ups are defined in a new ticket and justified
- [ ] Deviations from the acceptance criteria and design are agreed with the PO and documented.
- [ ] No debug or dead code
- [ ] My code has no repetitions
- [ ] Critical parts are tested automatically
- [ ] Where possible E2E tests are implemented
- [ ] Documentation/examples are up-to-date
- [ ] All non-functional requirements are met
- [ ] Functionality of the acceptance criteria is checked manually on the dev system.

### CODE REVIEW

File: .github/workflows/compile.yml [hunk 1]
```diff
@@ -87,7 +87,7 @@ jobs:
         path: executables
     - 
       name: move files one folder up
-      run: mv */*.tar . && find . -type d -empty -delete
+      run: mv */*.tar.gz . && find . -type d -empty -delete
```
Comment (fforootd) [inline .github/workflows/compile.yml:90]: Why do we need this?

File: .releaserc.js [hunk 2]
```diff
@@ -2,6 +2,7 @@ module.exports = {
   branches: [
     { name: "next" },
     { name: "next-rc", prerelease: "rc" },
+    { name: "pipeline-upload-assets", prerelease: "ignore-me2" },
```
Comment (fforootd) [inline .releaserc.js:5]:
```suggestion
```

File: .releaserc.js [hunk 3]
```diff
@@ -11,30 +12,41 @@ module.exports = {
...
+            label: "checksums.txt",
+          }
         ],
+        draftRelease: true,
+        successComment: false,
+        failComment: false,
+        labels: false,
+        releasedLabels: false,
+        addReleases: false,
+        failTitle: false,
```
Comment (fforootd) [inline .releaserc.js:49]:
```suggestion
```

File: .github/workflows/compile.yml (hunk 1 above)
Comment (stebenz) [reply .github/workflows/compile.yml:90]: just changed the file ending, no idea why the command itself is in here to calculate the checksum

File: .releaserc.js (hunk 2 above)
Comment (stebenz) [reply .releaserc.js:5]: only for testing the .tar.gz change, will be removed shortly

File: .releaserc.js (hunk 3 above)
Comment (stebenz) [reply .releaserc.js:49]: only for testing the .tar.gz change, will be removed shortly

File: .github/workflows/compile.yml (hunk 1 above)
Comment (fforootd) [reply .github/workflows/compile.yml:90]: ok I see

**Review by fforootd — APPROVED** [review]

**Comment by fforootd** [comment]
replaces https://github.com/zitadel/zitadel/pull/6349

**Pull request merged**